tq42/functional_tests/invoke_cli/test_functional_experiment_group.py

```diff
@@ -17,7 +17,7 @@ def test_exp_list(self):
         assert result.output is not None
 
         # need a default set for just running exp list
-        Project(client=self.get_client(), id=self.proj).set()
+        Project(client=self.get_client(), id=self.proj).set_as_default()
```

Comment:
as said in the other comment not sure if we are actually consistent with the default phrase as we are diverging between local default and remote default...